app/controllers/devise_token_auth/unlocks_controller.rb

```diff
@@ -61,9 +61,14 @@ def render_create_error_missing_email
     end
 
     def render_create_success
+      message = if Devise.paranoid
```

Comment:
same, I see this pattern repeated multiple times, maybe we can create a method that for a I18n key, it returns the paranoid version if `Devise.paranoid` is present

Reply:
@MaicolBen thanks for the suggestion, I will do it.